admin/templates/users/spam.html

```diff
@@ -0,0 +1,85 @@
+{% extends "base.html" %}
```

Comment:
Would it make more sense for this file to be `/templates/spam/user.html`? We have a `/spam/comment.html` template, not `/comments/spam.html`.